source/fe/fe_trace.cc

```diff
@@ -49,6 +49,13 @@ FE_TraceQ<dim,spacedim>::FE_TraceQ (const unsigned int degree)
 
   // Initialize face support points
   this->unit_face_support_points = fe_q.get_unit_face_support_points();
+
+  // initialize unit support points (this makes it possible to assign initial
+  // values to FE_TraceQ)
+  this->unit_support_points.resize(this->dofs_per_cell);
+  for (unsigned int i=0; i<this->dofs_per_cell; ++i)
+    this->unit_support_points[i] = fe_q.get_unit_support_points()[i];
```

Comment:
Well, it does leave out the support points associated with the element-local degrees of freedom. For example, for Q3 elements in 2D `this->dofs_per_cell` is 12 and not 16 as in FE_Q, so the last 4 are left out. This is a subtle difference and probably worth a comment.